src/kytos/interfaceInfo.vue

```diff
@@ -19,6 +19,35 @@
           </k-property-panel>
       </k-accordion-item>
 
+      <k-accordion-item title="Metadata" v-if="Object.keys(this.metadata_items).length !== 0">
+         <div class="metadata_table">
+            <table>
+              <thead>
+                <tr>
+                  <th>Key</th>
+                  <th>Value</th>  
+                </tr>
+              </thead>
+              <tbody>
+                <tr v-for="(value, key) in this.metadata_items">
+                  <td >{{key}}</td>
+                  <td >{{value}}</td>
+                </tr>
+              </tbody>
+            </table>
+         </div>
+      </k-accordion-item>
+      <k-accordion-item title="Metadata actions"> 
```

Comment:
@Alopalao I think you'd have to adapt the CSS here, as it is, it stretched other components like the `k-property-panel-item` of the Basic Details, example:

![2022-04-08-151027_910x572_scrot](https://user-images.githubusercontent.com/1010796/162498405-dd9aa9fd-d2fd-45f5-83d6-b241b5cad8c5.png)

It's desirable to have something similar that you've implemented recently (notice that on the switch  it doesn't impact stretching other components):

![2022-04-08-151040_426x198_scrot](https://user-images.githubusercontent.com/1010796/162498505-660c8517-1fd9-4f91-b541-98b771b8e4db.png)

Could you adapt it accordingly? Thanks.

Reply:
So sorry again, I forgot to mention this is a branch from the UI PR #23 so it needs the topology PR [#85](https://github.com/kytos-ng/topology/pull/85)